.circleci/config.yml

```diff
@@ -0,0 +1,143 @@
+    keys:
+      - v1-yarn-cache-{{ checksum "yarn.lock" }}
+      - v1-yarn-cache
+
+  - &save_yarn_cache
+    name: 'Saving yarn cache'
+    key: v1-yarn-cache-{{ checksum "yarn.lock" }}
+    paths:
+      - ~/.cache/yarn
+
+executors:
+  node_11:
+    docker:
+      - image: circleci/node:11.10
+    working_directory: *working_directory
+
+orbs:
+  cypress: cypress-io/cypress@1
+
+commands:
+  install_dependencies:
+    description: 'Installing dependencies'
+    steps:
+      - checkout
+      - restore_cache: *restore_yarn_cache
+      - run: yarn install --pure-lockfile
+      - save_cache: *save_yarn_cache
+  build_artefacts:
+    description: 'Building artefacts'
+    steps:
+      - run: yarn build
+  lint:
+    description: 'Running linters'
+    steps:
+      - run: yarn run jest --projects jest.eslint.config.js jest.stylelint.config.js --maxWorkers=3 --reporters jest-silent-reporter
+  unit_test:
+    description: 'Running tests'
+    steps:
+      - run: yarn run jest --projects jest.test.config.js --maxWorkers=3 --reporters jest-silent-reporter
+  vrt_test:
+    description: 'Running Visual Regression Tests'
+    steps:
+      - run: yarn run-if:percy
+  build_playgrond:
+    description: 'Building playground'
+    steps:
+      - run: yarn playground:build
+  build_starter:
+    description: 'Building starter'
+    steps:
+      - run: yarn template-starter:build
+  publish_to_canary:
+    steps:
+      - install_dependencies
+      - build_artefacts
+      - run:
+          name: Publishing to npm registry
+          command: ./scripts/release_canary.sh
+
+jobs:
+  lint:
+    executor: node_11
+    steps:
+      - install_dependencies
+      - lint
+  e2e_test_playground:
+    executor: cypress-io/default
+    description: 'Running E2E tests on playground'
+    steps:
+      - cypress/run:
+          name: 'e2e-playground'
+          yarn: true
+          spec: 'cypress/integration/playground/**/*.js'
+          build: yarn playground:build
+          start: yarn run server-test playground:start:prod:local 3001 test:e2e:playground
+          wait-on: 'http://localhost:3001'
+          requires:
+            - cypress/install
+          store_artifacts: true
+          parallel: false
+          parallelism: 1
+  e2e_test_starter:
+    executor: cypress-io/default
+    description: 'Running E2E tests on starter'
+    steps:
+      - cypress/run:
+          name: 'e2e-starter'
+          yarn: true
+          spec: 'cypress/integration/template-starter/**/*.js'
+          build: yarn template-starter:build
+          start: yarn run server-test template-starter:start:prod:local 3001
+          wait-on: 'http://localhost:3001'
+          requires:
+            - cypress/install
+          store_artifacts: true
+          parallel: false
+          parallelism: 1
+  unit_test:
+    executor: node_11
+    steps:
+      - install_dependencies
+      - unit_test
+      - build_artefacts
+      - vrt_test
```

Comment:
We could think of having this running as a parallel step like for the e2e tests, it might speed up things a bit more

Reply:
I'd first like to have it running before optimizing.

Reply:
That said, I don't see splitting out VRTs gaining too much. The unit tests run fast and we can benefit from a "living" job (checked out code, built etc).

Reply:
I split them up anyway. Whatever.

Reply:
No, the apps are part of yarn workspace, to ensure that lerna updates the deps when releasing

Reply:
Ah missed that. Then I can merge the e2e and unit test workflows.